.ci/Jenkinsfile-integration-test

```diff
@@ -33,40 +33,65 @@ pipeline {
             }
         }
 
-        stage('Tests') {
+        stage('Code linting') {
```

Comment:
I think this can be replaced with the generic integration testing pipeline available in IRT: https://github.com/inmanta/irt/blob/master/ci/inmanta-tools/Jenkinsfile-integration-tests